CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Next release
 
+# 1.2.3
+This is a security release that will harden the application against CVE 2019-3465
+* Implement countermeasures against CVE 2019-3465 #138
```

Comment:
The PR title does not match the actual title, should be: `Upgrade xmlseclibs to version 3.0.4 #138`

Reply:
Done